pytorch_lightning/overrides/data_parallel.py

```diff
@@ -220,6 +220,11 @@ def reducer_prepare_for_backwards(self, output):
     def reducer_reset_hooks(self):
         self._reducer_prepared_for_backwards = False
 
+    @property
+    def running_single_process_per_device(self) -> bool:
```

Comment:
@awaelchli I think this is fine, in your refactor you've made this a wrapper module. We can move this logic to the wrapper right?  https://github.com/PyTorchLightning/pytorch-lightning/pull/5185/files#diff-d9eba1c616cf4db7a99b8fcdb0e97f9de7b92df25eba32ca0a22f51993735c57R155